package.json

```
@@ -1,6 +1,6 @@
 {
     "name": "node-red-admin",
-    "version": "0.2.6",
+    "version": "0.3.0",
```

Given we haven't added any new features with this change, I suggest its a fix-level change.

```suggestion
    "version": "0.2.7",
```